cypress/support/commands.js

```diff
@@ -23,3 +23,122 @@
 //
 // -- This is will overwrite an existing command --
 // Cypress.Commands.overwrite("visit", (originalFn, url, options) => { ... })
+
+Cypress.Commands.add("registryReady", (registryName) => {
+	waitUntilRegistryIsReady(registryName, 0);
+})
+
+Cypress.Commands.add("deleteRegistry", (registryName, apiKey) => {
+	deleteRegistry(registryName, apiKey);
+})
+
+Cypress.Commands.add("createEmptyRegistry", (registryName, apiKey, metadataFile) => {
+	createRegistry(registryName, apiKey, metadataFile, false);
+})
+
+Cypress.Commands.add("createNonEmptyRegistry", (registryName, apiKey, metadataFile) => {
+	createRegistry(registryName, apiKey, metadataFile, true);
+})
+
+Cypress.Commands.add("createEntity", (registryName, apiKey, dataFile) => {
+	createEntity(registryName, apiKey, dataFile);
+})
+
+function waitUntilRegistryIsReady(registryName, count){
+
+	let statusUrl = '/registry/' + registryName + '/status'
+	cy.log('waiting...')
+	cy.request({
+		url: statusUrl,
+		failOnStatusCode: false
+	}).then(function (response) {
+		if(response.status === 303){
+			const newCount = count + 1;
+			if(newCount < 5){
+				cy.wait(2000)
+				waitUntilRegistryIsReady(registryName, newCount)
+			}
+		}
+	})
+}
+
+// create registry
+function createRegistry(registryName, apiAdminApiKey, metadataFile, createEntity) {
+	cy.log('Using apiKey ' + apiAdminApiKey)
+	cy.fixture(metadataFile)
+	.then(function (testSchema) {
+		testSchema.id = registryName;
+		let createUrl = '/registry';
```

Comment:
Constants in case we change the api paths?

Reply:
Need to build urls with registrynames and entityids. Maybe for later